test-configs.yaml

```diff
@@ -1324,22 +1326,32 @@ test_configs:
   - device_type: qcom_qdf2400
     test_plans: [boot]
 
-  - device_type: qemu_arm
+  - device_type: qemu-arm-virt-gicv2
     test_plans:
       - boot_qemu
       - simple_qemu
-      - uefi-arm-gicv2
-      - uefi-arm-gicv3
+      - uefi-arm
       - v4l2_compliance_qemu_vivid
 
-  - device_type: qemu_arm64
+  - device_type: qemu-arm-virt-gicv3
     test_plans:
       - boot_qemu
       - simple_qemu
-      - uefi-arm64-gicv2
-      - uefi-arm64-gicv3
+      - uefi-arm
       - v4l2_compliance_qemu_vivid
 
+  - device_type: qemu-arm64-virt-gicv2
+    test_plans:
+      - boot_qemu
+      - uefi-arm64
```

Comment:
Please also keep the test plans in alphabetical order.

Reply:
ok